project/legislation/legislatorslocation.py

```diff
@@ -0,0 +1,35 @@
+import Model from model
+
+def get_legislator_and_district(account):
+    """
+    This first sanity checks the zip, then makes sure it is in the USA.  If yes, continue.
+    This tries to pull from our local database, and failing that, from Propublica.
+    """
+    
+    location = account.zip_code
+
+    if ((location < 10000)||(location > 100000)):
+        throw BadRequestException #custom error this
+    else:  #valid zip
+        #get state, and ward, then get congresscritter for ward
+        
+
+    #Getting it from the ProPublica page works while we are small, but there is a limit
+    #of 5000 requests per day, so we need to download, and store the info locally
+    #update it once per day
+
+    
+
+    return NULL;
+
+def get_legislator_ids_by_lat_long(latitude, longitude):
```

Comment:
What is the goal of this function? I.e. what aspect of the legislators will be matched against the latitude and longitude coordinates?

See if we can use a spatial data model for the data and [GeoDjango distance query](https://docs.djangoproject.com/en/2.0/ref/contrib/gis/db-api/#distance-queries) to retrieve nearby results.